docs/notes.md

```diff
@@ -0,0 +1,27 @@
+# Tracking the impact of changes
+
+Some changes in your organization, either internal or external to the environment, can impact your software delivery process. For example, changes in the team structure or membership, teams adopting new working processes, or major events affecting the system stability, might have a positive or negative impact on the performance of your teams.
+
+Pulse enables you to track and correlate the impact that specific changes might have on the performance of your teams using **notes** to signal actions or events that took place on a specific date.
+
+![Pulse notes](images/notes.png)
+
+Pulse shows your organization notes across the different dashboards - [Accelerate overview](metrics/accelerate.md), [Changes & Reviews](metrics/accelerate-changes.md), and [Lead & Cycle time](metrics/lead-cycle-time.md). This way you can compare each performance metric before and after a specific change.
+
+!!! note
+    -   Notes are global to the organization. Any member can add or delete any note.
+    -   Pulse dashboards show the latest 100 notes for the selected time range.
```

Comment:
@mrfyda I tweaked this sentence to include the 100 notes limitation.
@therightmandev Can you please confirm if this limitation is correctly described? Is Pulse showing the latest 100 notes for the selected time range?

Reply:
@therightmandev I'm closing this conversation, as you already confirmed this in [Slite](https://codacy.slite.com/app/docs/0xFeGj0kTi3GGX?threadId=0Nqi_FOYX70j61).